test/test_basic.py

```diff
@@ -90,28 +91,30 @@ def request(self, url, retries=0):
         self.fail(f"timeout reached trying to request {url}")
 
     def testBasic(self):
-        response = self.request(f'{URL_3SCALE}{config.ROUTE_API}/sessions/new')
+        response = self.request(f'{self.api_url}{config.ROUTE_API}/sessions/new')
         self.assertEqual(response.status, 200)
         sessionid = json.load(response)['id']
         self.assertIsInstance(sessionid, str)
 
+        # API URL is on the container host's localhost; translate for the container DNS
+        host = self.api_url.replace('localhost', 'host.containers.internal').replace('https:', 'wss:')
```

Comment:
host seems to be a weird variable name? websocket_url maybe?

If it's `host` I would expect it to be `host.containers.internal`

Reply:
Right, renamed to websocket_url.